infra/docker-evm-blockhain/Dockerfile

```diff
@@ -3,18 +3,47 @@ FROM ethereum/client-go:alltools-latest
 # Install required packages
 RUN apk add --no-cache curl jq
 
+# ADD . /app
 # Set the working directory
-WORKDIR /root
+# WORKDIR /app
 
 # Copy the password file
-COPY root-password.txt /root/root-password.txt
+COPY root-password.txt /app/root-password.txt
 
 # Copy the startup script
-COPY start.sh /root/start.sh
+COPY start.sh /app/start.sh
+
+# Set the proper permissions for the script
+RUN chmod -R 777 /app
+RUN chmod -R 777 /bin/sh
+RUN chmod -R 777 /bin/busybox
+
+# Set the proper permissions for the script
+# RUN chmod +x /app/start.sh
+
+RUN ls -la /app
+RUN ls -la /bin/sh
+RUN ls -la /bin/busybox
+
+# Create a user with a known UID/GID within range 10000-20000.
+# This is required by Choreo to run the container as a non-root user.
+RUN adduser \
+    --disabled-password \
+    --gecos "" \
+    --uid 10014 \
+    "choreo"
+
+RUN chown -R choreo:choreo /app
```

Comment:
```suggestion
```

AFAIR we have added this `chown` for debugging purposes.  let's keep this only if it is really necessary.

Reply:
Tested without running `chown -R choreo:choreo /app` command. container working as expected.